services/node_manager/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+#ignite configuration
+ignite.host=localhost
+ignite.port=10800
+ignite.key-store-path=keystore.jks
+ignite.key-store-password=123456
+ignite.trust-store-path=truststore.jks
+ignite.trust-store-password=123456
+## MULTIPART (MultipartProperties)
+# Enable multipart uploads
+#spring.servlet.multipart.enabled=true
```

Comment:
Regarding old comments left for PR (#185), are these configuration related to the upload file API to limit the file uploading? If so, those are good configuration. Otherwise, completely remove them.

Reply:
Yes, that is the purpose the configuration.

Reply:
Based on my comments above, these are good configuration and we should enable them on, instead of remove .

Reply:
They are reverted.